app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizard.js

```diff
@@ -126,8 +126,18 @@ class MappingWizard extends React.Component {
 
   goToStep(activeStepIndex) {
     const { activeStepIndex: currentStep } = this.state;
+    const targetStepProp = mappingWizardSteps[activeStepIndex];
+    const targetStepForm = this.props[targetStepProp];
 
-    if (currentStep !== 4) {
+    if (activeStepIndex === 4) {
+      this.setState(prevState => ({
+        activeStepIndex: prevState.activeStepIndex
+      }));
```

Comment:
If clicking on this doesn't allow nav, please remove the pointer cursor from the step indicator for this step.